internal-metrics-monitoring/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.9'
+
+services:
+  vector:
+    image: timberio/vector:latest-alpine
+    environment:
+      - LOG=info
+    networks:
+      - backend
+    volumes:
+      - ./vector:/etc/vector:ro
+
+  prometheus:
+    image: bitnami/prometheus:latest
+    networks:
+      - backend
+    ports:
+      - 9090:9090
+    volumes:
+      - ./prometheus/prometheus.yml:/opt/bitnami/prometheus/conf/prometheus.yml:ro
+
+networks:
```

Comment:
This isn't blocking, but in general I feel like we should keep the Docker Compose configurations as simple as possible. In this case, Compose automatically creates a `default` network, which makes an explicit network like this unnecessary.